plenum/common/ledger_uncommitted_tracker.py

```diff
@@ -20,14 +21,25 @@ def apply_batch(self, state_root, ledger_size):
         if ledger_size <= 0:
             raise PlenumValueError('ledger_size',
                                    ledger_size,
-                                   "Incorrect size of ledger given")
+                                   "Incorrect size of ledger")
 
         self.un_committed.append((state_root, ledger_size))
 
     def commit_batch(self):
```

Comment:
It looks like we set `last_committed ` from the outside.
Why not setting it in `commit_batch` as an argument?

Reply:
We also need to support initial set of committed state (so that we can set it even if `un_committed` is empty).